playground/app/components/todo-item.component.ts

```diff
@@ -12,9 +12,19 @@ export class TodoItemCmp implements OnChanges{
 
   @Input('<') todo: TodoModel;
   @Input('<') idx: number;
-  @Output() onDone: ( todo: {todo:TodoModel} )=>void;
+  // this is old way how to bindings are created, will be removed in 2.0
+  // @Output() onDone: ( todo: {todo:TodoModel} )=>void;
+  @Output() onDone: EventEmitter<TodoModel>;
+  @Output() emitTest = new EventEmitter<string>();
 
-  ngOnInit(){}
+  ngOnInit(){
+
+    const dispose = this.onDone.subscribe( (value)=> {
+      console.log( `onDone emitted value: ${value}` );
```

Comment:
you don't have to subscribe, that's just showcase that subscribe works for EventEmmiter, and its registered "observer/callback" will be called when it's emitted via `this.emit('booo')`